### PR TITLE
chore: hint link color and text size

### PR DESCRIPTION
**Pull request by zhouhanseng**
close https://github.com/DimensionDev/Maskbook/issues/1210

image

image

### CODE REVIEW

**Review by Tedko — APPROVED** [review]

**Comment by Tedko** [comment]
is this part bug?

image

**Comment by zhouhanseng** [comment]
No, this was a demand

> Yes the font-size can be slightly larger when the text is short enough. It is a good practice employed by Facebook. And each SNS may have its predefined colors for links. We may want to have a site-specific strategy for managing these behaviors.

**Comment by Tedko** [comment]
ok!

**Comment by zhouhanseng** [comment]
@Tedko Approve again？

**Review by Tedko — APPROVED** [review]

**Comment by Tedko** [comment]
Look good for me

File: packages/maskbook/src/components/InjectedComponents/AdditionalPostContent.tsx [hunk 1]
```diff
@@ -72,7 +72,7 @@ export const AdditionalContent = React.memo(function AdditionalContent(props: Ad
             <header className={classes.content}>{header}</header>
             {message ? (
                 <main className={classes.content}>
-                    <DefaultTypedMessageRenderer {...props} message={TypedMessage} />
+                    <DefaultTypedMessageRenderer {...props} message={TypedMessage} allowTextEnlarge={true} />
```
Comment (Jack-Works) [inline packages/maskbook/src/components/InjectedComponents/AdditionalPostContent.tsx:75]: This design doesn't look right for me. I'll address an issue for it.

**Pull request merged**